lib/HackTest.rb

```diff
@@ -32,7 +40,7 @@ def search_files(files)
       file_name = file_hash[:filename]
       search_lines(non_inclusive_words_found, lines, file_name)
     end
-    non_inclusive_words_found
+    return non_inclusive_words_found
```

Comment:
Any reason for adding this? It's my understanding that Ruby doesn't conventionally use the `return` keyword. http://ruby-for-beginners.rubymonstas.org/writing_methods/return_values.html

Reply:
bookmarking this 😂